modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,45 @@
+# Import needed libraries
+from PIL import Image  # image libraries and operations
+import os, sys  # directory libraries
+import imghdr  # image file recognition
+import numpy  # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+# list of image types
+imgfiles = ['rgb', 'gif', 'pbm', 'pgm', 'ppm', 'tiff', 'rast', 'xbm', 'jpeg', 'jpg', 'bmp', 'png', 'webp', 'exr']
+
+
+# Splitting up this module into smaller ones
+
+# Given a filepath and size, resizeFolder will enter the folder
+# and edit the files to be square, with lxw as sizexsize
+def resizeFolder(path, size):
```

Comment:
I was thinking that too. I tried taking the folder-opening out of the modules, but I found that I would still need to feed the same filepath as a string into both modules, which kinda defeats the purpose of creating a new function to handle that stuff.

For now, I'm going to keep it as is, but more help with this would be appreciated